src/Talis/Persona/Client/Base.php

```diff
@@ -55,40 +57,52 @@
     private $defaultTtl;
 
     /**
+     * @var string
+     */
+    private $appUserAgent;
+
+    /**
+     * @var string
+     */
+    private $phpVersion;
+
+    /**
      * Constructor
      *
      * @param array $config An array of options with the following keys: <pre>
      *      persona_host: (string) the persona host you'll be making requests to (e.g. 'http://localhost')
      *      persona_oauth_route: (string) the token api route to query ( e.g: '/oauth/tokens')
+     *      userAgent: Consuming application user agent string @since 2.0.0
+     *            examples: rl/5.2, rl, rl/5, rl/5.2 (php/5.3; linux/2.5)
```

Comment:
Hmm, this wouldn't be something like `rl/{rl-release-sha}` (or whatever they use for release ids)?